lsstsal/scripts/gensimplesample.tcl

```diff
@@ -523,11 +523,11 @@ global OPTIONS
      puts $fout "	struct ackcmd \{"
      add_private_idl $fout
      puts $fout "	  long	ack;
-	  long	error;
+	  long		error;
```

Comment:
Can you please revert formatting changes?

Reply:
These formatting changes look fine in the context of the whole file.  I'd say leave them.